src/components/pages/Calendar.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react';
+import FullCalendar from '@fullcalendar/react';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+
+import * as constants from '../../constants';
+
+export default function Calendar(props) {
+  const { isAdmin } = props;
+
+  const [addEventText, setAddEventText] = useState('Add Event');
+  const [headerText, setHeaderText] = useState('today,prev,next,addEventButton');
+
+  const calendarRef = React.createRef();
+  // For Reactiveness
+  const updateMedia = () => {
```

Comment:
This looks really good when testing on desktop when using the responsive display. However, when I used my phone to test this, for some reason it defaulted to a day view with the desktop header. I have no idea how to even start solving this bug though so if you want to ignore it for now I guess we can.
![IMG_0671](https://user-images.githubusercontent.com/32073638/108607715-f04f5c00-7376-11eb-8e93-1ef90c5f5344.PNG)

Reply:
I was able to fix this, by calling updateMedia in the useEffect